packages/contract-wrappers/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@0x/contract-wrappers",
-    "version": "4.1.3",
+    "version": "4.2.0",
```

Comment:
Do we ever need to manually update versions in package.json?

Reply:
Just double-checked and we do not. Removed.